dallinger/config.py

```diff
@@ -6,6 +6,7 @@
 import imp
 import logging
 import os
+from sets import Set
```

Comment:
The rationale is that, otherwise, the `set` in `synonyms=set()` is interpreted as the new `config.set()` function instead of the built-in `set`, and I don't know of a way around it. Do you? (Also, we aren't compatible with Python 3 and won't be for a while because the MTurk API doesn't yet exist in `boto3`.)

```
================================================================== ERRORS ===================================================================
___________________________________________________ ERROR collecting tests/test_config.py ___________________________________________________
tests/test_config.py:9: in <module>
    from dallinger.config import Configuration
dallinger/__init__.py:3: in <module>
    from . import (
dallinger/command_line.py:25: in <module>
    from dallinger.config import get_config
dallinger/config.py:66: in <module>
    class Configuration(object):
dallinger/config.py:137: in Configuration
    def register(self, key, type_, synonyms=set(), sensitive=False):
E   TypeError: set() takes exactly 3 arguments (0 given)
```